virttest/qemu_vm.py

```diff
@@ -4214,19 +4214,26 @@ def _shell_reboot(session, timeout):
         error_context.context()
 
         start_time = time.time()
-        if method == "shell":
+        if (self.params.get("force_reset_go_down_check") == "qmp" and
+                isinstance(self.monitor, qemu_monitor.QMPMonitor)):
+            _check_go_down = partial(self.monitor.get_event, "RESET")
```

Comment:
Should be `_check_go_down = _go_down_qmp`

Reply:
Oups, copied it from a wrong patch. Also I introduced an undefined variable, let me take care of these...